tests/unit/views/Samples.spec.js

```diff
@@ -63,39 +68,59 @@ describe('Samples.vue', () => {
         response = {status: 201}
         samples.tractionApiLibrary.data = response
         samples.tractionApiLibrary.create.mockReturnValue(response)
-        samples.createLibrariesInTraction()
+
+        let selectedEnzymeId = 1
+        samples.createLibrariesInTraction(selectedEnzymeId)
         await flushPromises()
 
-        let sample_ids = []
+        let libraryAttrs = []
```

Comment:
Can we add a TODO to have a look at the processing in the test and see if it signifies an issue elsewhere.